package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, Luthaf, hmacdope, rafaelpap, jbarnoud
+09/16/22 IAlibay, Luthaf, hmacdope, rafaelpap, jbarnoud, aya9aladdin
```

Comment:
```suggestion
??/??/?? IAlibay, Luthaf, hmacdope, rafaelpap, jbarnoud, aya9aladdin
```

Version 2.4.0 is not yet released and we do not know when it will be. Hence the question marks instead of a date.